tests/dockerize.py

```diff
@@ -0,0 +1,23 @@
+# Prerequisites: Start a local Docker image with Runhouse installed using the following commands:
+#   $ docker build --pull --rm -f \"Dockerfile\" \
+#       --build-arg DOCKER_USER_PASSWORD_FILE=docker_user_passwd \
+#       -t runhouse:start .
+#   $ docker run --rm --shm-size=3gb -it -p 50052:50052 -p 6379:6379 -p 52365:52365 -p 22:22 \
+#       runhouse:start
+print("Starting Runhouse in a local Docker cluster")
+import runhouse as rh
+
+print("Runhouse version: ", rh.__version__)
+rh.configs.disable_data_collection()  # Workaround until we remove the usage of GCSClient from our code
+port = 50052
+c = rh.cluster(
+    name="local-docker", host=f"localhost:{port}", ssh_creds={"ssh_user": "root"}
+)
+print("Cluster created")
+c.up_if_not()
+print("Cluster is up: ", c.is_up())
```

Comment:
can we turn this into a proper unittest and use asserts to check for expected values?

Reply:
Done in https://github.com/run-house/runhouse/blob/b1ae55ed21c9049c9c5a7530d2c556dfe7b3883a/tests/test_http_server.py